client/src/client.rs

```diff
@@ -236,12 +236,40 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 		self.backend.state_at(*block)
 	}
 
-	/// Given a `BlockId` and a key prefix, return the matching child storage keys in that block.
+	/// Given a `BlockId` and a key prefix, return the matching storage keys in that block.
 	pub fn storage_keys(&self, id: &BlockId<Block>, key_prefix: &StorageKey) -> sp_blockchain::Result<Vec<StorageKey>> {
 		let keys = self.state_at(id)?.keys(&key_prefix.0).into_iter().map(StorageKey).collect();
 		Ok(keys)
 	}
 
+	/// Paged version of `storage_keys`. Only return up to `count` keys after `start_key` or `prefix`.
+	pub fn storage_keys_paged(
+		&self,
+		id: &BlockId<Block>,
+		prefix: &StorageKey,
+		count: u32,
```

Comment:
I think it should be `usize` in this layer already.
```suggestion
		count: usize,
```